packages/plugin-ext/src/hosted/node/hosted-plugin-process.ts

```diff
@@ -156,9 +159,11 @@ export class HostedPluginProcess implements ServerPluginRunner {
             logger: this.logger,
             args: []
         });
-        this.childProcess.on('message', message => {
+
+        this.messagePipe = new BinaryMessagePipe(this.childProcess.stdio[4] as Duplex);
```

Comment:
fyi: The `BinaryMessagePipe` class currently uses a `net.Socket` to wrap the file descriptor, but this doesn't seem to work properly on Windows and leads to communication issues.